qemu/tests/virtio_fs_share_data.py

```diff
@@ -555,6 +558,36 @@ def __file_check(file, guest=None):
                                       "host and guest are inconsistent")
                         session.cmd("cd -")
                     else:
+                        def __file_check(sym_link, guest=None):
```

Comment:
@xiagao I think we can make def __file_check() support both Linux and win
Simple example:
```
        def __file_check(file, os_type, guest=None):
            cmd_map = {'win_host':'cat %s',
                           'win_guest':'type %s',
                           'linux_host' ='ls -l %s',
                           'linux_guest' ='ls -l %s'}
            
            cmd = '_'.join((os_type, guest))
            if guest:
                o = session.cmd_output(cmd_map[cmd] % file)
            else:
                o = process.run(cmd_map[cmd] % file).stdout_text
            return o.strip().split()[-1]
```

@YongxueHong What do you think?

Reply:
Hi @zhenyzha 
yeah, it's okay for me if we are gonging to support the Linux guest as well.

Reply:
@zhenyzha Updated, could you review it again？